harness.sh

```diff
@@ -0,0 +1,260 @@
+--listen=127.0.0.1:19559 --rpclisten=127.0.0.1:19560 \
+--miningaddr=${BETA_MINING_ADDR} \
+--txindex --connect=127.0.0.1:18555 \
+--debuglevel=debug \
+--simnet" C-m
+
+# ###############################################################################
+# Setup the beta node's dcrctl (beta-ctl).
+################################################################################
+sleep 3
+cat > "${NODES_ROOT}/beta/mine" <<EOF
+#!/bin/sh
+  NUM=1
+  case \$1 in
+      ''|*[!0-9]*)  ;;
+      *) NUM=\$1 ;;
+  esac
+  for i in \$(seq \$NUM) ; do
+    dcrctl -C beta-ctl.conf generate 1
+    sleep 0.5
+  done
+EOF
+chmod +x "${NODES_ROOT}/beta/mine"
+
+tmux new-window -t $SESSION:5 -n 'beta-ctl'
+tmux send-keys "cd ${NODES_ROOT}/beta" C-m
+echo "Mining 30 blocks"
+tmux send-keys "./mine 30" C-m
+sleep 10
+echo "At Stake Enabled Height (SEH)"
+
+
+################################################################################
+# Setup the beta wallet (w-beta).
+################################################################################
+cat > "${NODES_ROOT}/w-beta/ctl" <<EOF
+#!/bin/sh
+dcrctl -C w-beta-ctl.conf --wallet \$*
+EOF
+chmod +x "${NODES_ROOT}/w-beta/ctl"
+
+tmux new-window -t $SESSION:6 -n 'w-beta'
+tmux send-keys "cd ${NODES_ROOT}/w-beta" C-m
+tmux send-keys "dcrwallet -C w-beta.conf --create" C-m
+echo "Creating simnet beta wallet"
+sleep 1
+tmux send-keys "${WALLET_PASS}" C-m "${WALLET_PASS}" C-m "n" C-m "y" C-m
+sleep 1
+tmux send-keys "${BETA_WALLET_SEED}" C-m C-m
+tmux send-keys "dcrwallet -C w-beta.conf --debuglevel=debug" C-m
+
+################################################################################
+# Setup the beta wallet's dcrctl (w-beta-ctl).
+################################################################################
+sleep 1
+tmux new-window -t $SESSION:7 -n 'w-beta-ctl'
+tmux send-keys "cd ${NODES_ROOT}/w-beta" C-m
+tmux send-keys "./ctl getbalance"
+
+################################################################################
+# Setup the beta wallet's dcrctl (w-beta-ctl).
+################################################################################
+sleep 5
+cat > "${NODES_ROOT}/harness-ctl/reorg" <<EOF
+#!/bin/sh
+echo "Disconnecting beta from alpha"
+sleep 1
+tmux send-keys -t $SESSION:5 "./ctl addnode 127.0.0.1:18555 remove" C-m
+echo "Mining a block on alpha"
+sleep 1
+tmux send-keys -t $SESSION:1 "./mine 1" C-m
+echo "Mining 3 blocks on beta"
+tmux send-keys -t $SESSION:5 "./mine 3" C-m
+sleep 2
+echo "Reconnecting beta to alpha"
+tmux send-keys -t $SESSION:5 "./ctl addnode 127.0.0.1:18555 add" C-m
+sleep 2
+grep REORG ~/$SESSION/alpha/logs/simnet/dcrd.log
+EOF
+chmod +x "${NODES_ROOT}/harness-ctl/reorg"
+
+tmux new-window -t $SESSION:8 -n 'harness-ctl'
+tmux send-keys "cd ${NODES_ROOT}/harness-ctl" C-m
+tmux send-keys "./reorg"
+
+echo "Mining 112 blocks"
+tmux send-keys -t $SESSION:5 "./mine 112" C-m
+sleep 70
+echo "At Stake Validation Height (SVH)"
+sleep 5
+
```

Comment:
Should we mine an additional `CoinbaseMaturity` above `StakeValidationHeight` so that the wallet will have some valid stakebase outputs?